AgentDiscoveries-Frontend/app/src/components/app.jsx

```diff
@@ -25,7 +25,8 @@ export default class App extends React.Component {
             <React.Fragment>
                 <Router>
                     <Switch>
-                        <Route path='/' exact render={() => <Page><Home /></Page>} />
+                        <Route path='/welcome' exact render={() => <Page><Home /></Page>} />
+                        <Route path='/' exact render={() => <Page><Login /></Page>} />
```

Comment:
Do we want '/' to show the same page as '/login' or do we just want '/' to redirect if you're not logged in?